mdata/cache/accnt/if.go

```diff
@@ -20,3 +20,8 @@ type EvictTarget struct {
 	Metric string
 	Ts     uint32
 }
+
+func NewAccnt(maxSize uint64) Accnt {
+	// currently we only know NewFlatAccnt
+	return NewFlatAccnt(maxSize)
+}
```

Comment:
Ah, so we can use `NewFlatAccnt()` in the tests. If we'd only have one `New*` method to instantiate it, which returns an interface, then we'd have to choose if in the tests we want to manually instantiate it or not access any of the private variables.